node/src/archive/sqlite.rs

```diff
@@ -119,6 +132,24 @@ impl Archive {
 
         Ok(serde_json::from_str(&events.json_contract_events)?)
     }
+
+    /// Check if a block_height & block_hash match a finalized block
+    pub async fn match_finalized_block_height_hash(
```

Comment:
Include an `# Error` rust doc on top of this message and document when it can fail (if its exported and used by a consumer)

Reply:
This function can only fail if the db fails which would be an internal server error.